src/libasr/ASR.asdl

```diff
@@ -87,7 +87,7 @@ expr
     | IntrinsicElementalFunction(int intrinsic_id, expr* args, int overload_id, ttype? type, expr? value)
     | IntrinsicArrayFunction(int arr_intrinsic_id, expr* args, int overload_id, ttype? type, expr? value)
     | IntrinsicImpureFunction(int impure_intrinsic_id, expr* args, int overload_id, ttype? type, expr? value)
-    | TypeInquiry(int inquiry_id, ttype arg_type, expr? arg, ttype type, expr value)
+    | TypeInquiry(int inquiry_id, ttype arg_type, expr? arg, ttype type, expr? value)
```

Comment:
This change is not correct actually. `expr?` means a field which is optional and can be set to `nullptr`. Here, `expr? value` refers to compile-time value, which can be absent sometimes.